library/composer.json

```diff
@@ -6,6 +6,7 @@
         "h4cc/wkhtmltopdf-amd64": "0.12.3",
         "h4cc/wkhtmltopdf-i386": "0.12.3",
         "crada/php-apidoc":"@dev",
-        "xamin/handlebars.php": "^0.10.4"
+        "xamin/handlebars.php": "^0.10.4",
+        "debach/zend-mp3": "dev-master"
```

Comment:
Even though this library hasn't been updated in a while I would point to 1.8.1 instead of dev-master